static/app/components/onboarding/gettingStartedDoc/types.ts

```diff
@@ -48,6 +48,10 @@ export interface DocsParams<
   projectSlug: Project['slug'];
   sourcePackageRegistries: {isLoading: boolean; data?: ReleaseRegistrySdk};
   cdn?: string;
+  feedbackOptions?: {
+    email?: boolean;
+    name?: boolean;
```

Comment:
nit: slightly longer names would clarify what it is used for 🙂 
```suggestion
    isEmailRequired?: boolean;
    isNameRequired?: boolean;
```